Exercises/filling2.cpp

```diff
@@ -34,12 +34,27 @@ bool PathDecomposition::Check() {
 
 	for (int i = 0; i < n; ++i) {
 		if (sum[i] == 0) throw PathDecomposition::CorectnessException("Not all vertices are in path-width decomposition");
-		if (right[i] - left[i] + 1 != sum[i]) throw PathDecomposition::CorectnessException("In Decomposition exists vertex u and i < j < k such that bug[i] and bug[k] contains u but bug[j] doesn't");
+		if (right[i] - left[i] + 1 != sum[i]) {
+			int j = left[i];
+			for (; 
+				std::find(_bugs[j].begin(), _bugs[j].end(), i) != _bugs[j].end(); 
+				++j);
+			std::string msg = "In Decomposition _bugs["
+				+ std::to_string(left[i])
+				+ "] and _bugs["
+				+ std::to_string(right[i])
+				+ "] contains "
+				+ std::to_string(i)
+				+ " but _bugs["
+				+ std::to_string(j)
+				+ "] does not";
+			throw PathDecomposition::CorectnessException(msg);
```

Comment:
Maybe it makes sense to save this info in the exception explicitly and then access it from unit tests?